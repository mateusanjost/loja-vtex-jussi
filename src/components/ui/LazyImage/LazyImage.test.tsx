import TestRenderer from "react-test-renderer";
import LazyImageComponent from "./index";

describe("LazyImage", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<LazyImageComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
