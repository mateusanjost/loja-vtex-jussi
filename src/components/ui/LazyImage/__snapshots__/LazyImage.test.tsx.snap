// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LazyImage should match snapshot 1`] = `
<div
  className="lazyload-wrapper "
>
  <div
    className="lazyload-placeholder"
    style={
      Object {
        "height": undefined,
      }
    }
  />
</div>
`;
